### DISCUSSION TITLE
Will this method be added to Nerfstudio?

### DISCUSSION THREAD

**Discussion by marcocentin** [hbb1/2d-gaussian-splatting#89] [Q&A] [answered]
Hello everyone, I wonder if you guys are looking into adding this method to Nerfstudio anytime soon to allow people to easily test your results and compare.

**Comment by hbb1** [comment, marked as answer]
Hi, I am currently overwhelmed by some stuff. Luckily,  a guy has made some progress on implementing 2DGS based on gsplat https://github.com/nerfstudio-project/gsplat/pull/208

**Reply by marcocentin** [reply]
Thanks for the info and for the quick reply!
